mapactionpy_arcmap/arcmap_runner.py

```diff
@@ -126,28 +140,79 @@ def getOrientation(countryName):
         print("Error: Could not derive country extent from " + url)
         print("Exiting.")
         sys.exit(1)
-           
+
 
 def main(args):
     args = parser.parse_args()
-    cookbookFile = args.cookbookFile
-    layerPropertiesFile = args.layerConfig
+
+    # Construct a Crash Move Folder object if the cmf_description.json exists
     crashMoveFolder = args.crashMoveFolder
-    layerDirectory = args.layerDirectory
+    cmfFilePath = os.path.join(crashMoveFolder, "cmf_description.json")
+    eventFilePath = os.path.join(crashMoveFolder, "event_description.json")
+
+    cmf = None
+    event = None
+
+    if os.path.exists(cmfFilePath):
+        cmf = CrashMoveFolder(cmfFilePath)
+        event = Event(eventFilePath)
+
     productName = args.productName
-    countryName = args.countryName
 
+    # If no country name supplied, need to find it from the event_description.json
+    if args.countryName:
+        countryName = args.countryName
+    else:
+        if event is not None:
+            country = pycountry.countries.get(alpha_3=event.affected_country_iso3.upper())
+            countryName = country.name
+        else:
+            print("Error: Could not derive country from " + eventFilePath)
+            print("Exiting.")
+            sys.exit(1)
```

Comment:
LN 162 to 202:
With all of the `sys.exit()` calls here, I suggest that we have a bunch of constants defined somewhere like this: 
```
ERROR_EVENT_DESCRIPTION_FILE_NOT_FOUND = 2
ERROR_CMF_DESCRIPTION_FILE_NOT_FOUND = 4
```
Then these should be used asthe exit codes, so that the calling program might identify the cause of the problem.
(PS. I don't know what the PEP8 standard is for this type of property)